lib/version.c

```diff
@@ -73,6 +73,14 @@
 #include <gsasl.h>
 #endif
 
+#ifdef USE_NGTCP2
+#include <ngtcp2.h>
+#endif
+
+#ifdef USE_NGHTTP3
+#include <nghttp3.h>
+#endif
```

Comment:
This is not how these header files are included. They are located in subdirectories. But I also have a more generic thought on this.

Reply:
I am not entirely sure how to handle this. If USE_NGHTTP3 defined, the include folders point to the folder with this header file in it. Also, it is done similarly for other library header files, too. So I am not exactly sure what to do. But I take your second part of the remark as "I have a better idea" - which is totally fine with me. So this is just to confirm I read your comment but will/can nothing do about without further advise. :-)

Reply:
1. The function that outputs the quic and http3 versions should be in the `curl_ngtcp2.c` file, and correspondingly in `curl_msh3.c`  and `curl_quiche.c.
2. In `vquic.c` you create the generic quic function(s) that is called from version.c to output the version

See the TLS library version as an example.